lib/auth/session.ts

```
@@ -0,0 +1,20 @@
+import { SessionOptions } from 'iron-session';
+
+export interface SessionData { 
+    username: string;
+    password:string;
+}
+
+export const sessionOptions: SessionOptions = {
+  cookieName: 'session',
+  password: process.env.SESSION_SECRET!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)